mapper/pojo-standalone/src/main/java/org/hibernate/search/mapper/pojo/standalone/mapping/SearchMapping.java

```diff
@@ -126,18 +127,22 @@ default <T> SearchScope<T> scope(Class<T> expectedSuperType, String entityName)
 	Backend backend(String backendName);
 
 	/**
+	 * @param annotatedTypeSource A source of types to be processed for annotations by Hibernate Search.
 	 * @return A {@link SearchMapping} builder.
+	 * @see AnnotatedTypeSource
 	 */
-	static SearchMappingBuilder builder() {
-		return new SearchMappingBuilder();
+	static SearchMappingBuilder builder(AnnotatedTypeSource annotatedTypeSource) {
```

Comment:
Removing `builder()` (no param) was the whole point :)

I want the API to make it very clear that Hibernate Search needs some input to be able to process annotations. Otherwise that's too easily missed.

Those who want only programmatic mapping will have pass `AnnotatedTypeSource.empty()` explicitly.